lib/jsonapi_parameters/default_handlers/base_handler.rb

```diff
@@ -19,6 +19,10 @@ def find_included_object(related_id:, related_type:)
                 included_object_enum[:type] == related_type
             end
           end
+
+          def self.call(key, val, included)
```

Comment:
I would put class methods at the top of the class definition

Reply:
Good idea!